images/gotools/bin/build-protoc

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
is this file needed? I don't see any reference elsewhere.

Reply:
This file has been retained in case we will revert back to using the alpine version of the go image (since there are no pre-compiled binaries available for that system).